### PR TITLE
Add Client ID to send_verification_email

### LINKED ISSUE #92: Client specific post email verification redirect URL problem in V2
Greetings fellow Auth0 geeks,

I have an existing Auth0 account with multiple clients and [conditional post email verification redirects](https://auth0.com/docs/email/custom#custom-redirect) specified through the Auth0 dashboard.
When triggering a resend verification email job via the Ruby V2 client, the redirect to URL points to the default URL, not the one specified for the given `client_id`.

This is not the behavior in the V1 implementation where the `client_id` is correctly applied and the redirect follows to the specified URL.

Looking at the docs for [Send a "verify email address" email](https://auth0.com/docs/api/management/v2#!/Jobs/post_verification_email) a `client_id` body attribute is accepted, where as [the Ruby implementation](https://github.com/auth0/ruby-auth0/blob/master/lib/auth0/api/v2/jobs.rb#L42-L49) (to my understanding) does not apply the `client_id`:

```ruby
def send_verification_email(user_id)
  raise Auth0::InvalidParameter, 'Must specify a user id' if user_id.to_s.empty?
  request_params = {
    user_id: user_id
  }
  path = "#{jobs_path}/verification-email"
  post(path, request_params)
end
```

Naively, should we add an optional `client_id` argument and add it to the `request_params` if the value is present?

Thanks for your feedback

### PR DESCRIPTION
**Pull request by joshcanhelp**
### Changes

- Add Add Client ID to `Auth0::Api::V2::Jobs.send_verification_email`
- Remove skipped Jobs tests and add VCR cassettes

### References

Fixes #92 

### Testing

[Failing tests for new feature](https://travis-ci.org/auth0/ruby-auth0/builds/499524045)

* [x] This change adds unit test coverage
* [x] This change adds integration test coverage
* [x] This change has been tested on the latest version of Ruby (2.5.1)

### Checklist

* [x] All existing and new tests complete without errors
* [x] Rubocop passes on all added/modified files
* [x] All active GitHub checks have passed

### CODE REVIEW

File: lib/auth0/api/v2/jobs.rb [hunk 1]
```diff
@@ -76,11 +79,12 @@ def jobs_path
...
         # Map array of field names for export to array of objects
-        # @param fields [array] Field names to be included in the export
-        
-        # @return [array] Returns the fields mapped as array of objects for the export_users endpoint
+        # @param fields [array] Field names to be included in the export
+
+        # @return [array] Returns the fields mapped as array of objects for the export_users endpoint
         def fields_for_export(fields)
-          return nil if fields.nil? || fields.empty?
+          return nil if fields.to_s.empty?
```
Comment (luisrudge) [inline lib/auth0/api/v2/jobs.rb]: why the change here?

File: lib/auth0/mixins/httpproxy.rb [hunk 2]
```diff
@@ -21,6 +21,7 @@ module HTTPProxy
                    elsif method == :delete
                      call(:delete, url(safe_path), timeout, add_headers({params: body}))
                    elsif method == :post_file
+                     body.merge!(multipart: true)
```
Comment (luisrudge) [inline lib/auth0/mixins/httpproxy.rb]: why?

File: lib/auth0/api/v2/jobs.rb (hunk 1 above)
Comment (joshcanhelp) [reply lib/auth0/api/v2/jobs.rb]: Simplification ... I think it was suggested by the formatting tool `rubocop`

File: lib/auth0/mixins/httpproxy.rb (hunk 2 above)
Comment (joshcanhelp) [reply lib/auth0/mixins/httpproxy.rb]: The import method was not working as it was written, which I found out when the tests were finally running (instead of being skipped).

Comment (joshcanhelp) [reply lib/auth0/mixins/httpproxy.rb]: The import method was not working as it was written, which I found out when the tests were finally running (instead of being skipped).
